tests/unit/CertificateGenerationService.unitTest.ts

```diff
@@ -1031,7 +1031,7 @@ describe("Certificate Generation Service", () => {
             const event = cloneDeep(queueEventPass);
             const testResult: any = JSON.parse(event.Records[0].body);
 
-            it("should allow PASS test result for Welsh translation", async () => {
```

Comment:
Out of interest, why can't these be async any more?

Reply:
It's not using any async functionality inside of the test, so I don't think that it's required to be async